bitnami/thanos/values.yaml

```diff
@@ -2290,6 +2290,97 @@ receive:
   ## @param receive.enabled Enable/disable Thanos Receive component
   ##
   enabled: false
+  ## @param Mode to run receiver in. Valid options are "standalone" or "dual-mode"
+  ## ref: https://github.com/thanos-io/thanos/blob/release-0.22/docs/proposals-accepted/202012-receive-split.md
+  ## Thanos receiver mode. Setting this to "dual-mode" will create a deployment for the stateless thanos distributor.
+  mode: standalone
+
+  distributor:
+    ## Thanos Receive Distributor containers' resource requests and limits
+    ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+    ## We usually recommend not to specify default resources and to leave this as a conscious
+    ## choice for the user. This also increases chances charts run on environments with little
+    ## resources, such as Minikube. If you do want to specify resources, uncomment the following
+    ## lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+    ## @param receive.distributor.resources.limits The resources limits for the Thanos Receive container
+    ## @param receive.distributor.resources.requests The requested resources for the Thanos Receive container
+    ##
+    resources:
+    ## Example:
+    ## limits:
+    ##    cpu: 100m
+    ##    memory: 128Mi
+    ##
+    limits: {}
```

Comment:
`limits` and `requests` here seem to be properties of `resources` and hence should have one more level of indentation

Reply:
yep good catch, fixed